src/js/components/TabPanel/InfoWindow.js

```diff
@@ -239,6 +241,43 @@ export default class InfoWindow extends Component {
             featuresList: [feature]
           };
         }
+      } else {
+        if (feature._layer && feature._layer.layerId) {
+          let id = feature._layer.id;
+          if (id === 'PA_4') {
```

Comment:
@KaylaKremer We can't keep these specific **layerId** hotfixes in here!

Reply:
@SwampGuzzler Just a bit confused about what you pointed out for DRC. My build I made for this PR uses Cameroon. I made two additional builds, one for DRC, but that was for `#628`.

And yes, wanted to discuss this in regards to the `layerId` fixes since we have not addressed this yet for Georgia! If you'd like to pair program this sometime today, let me know!